app/views/users/show.html.erb

```diff
@@ -2,10 +2,11 @@
 <html>
 <head>
   <title>User Details</title>
+  <link rel="stylesheet" type="text/css" href="/assets/users.css"> <!-- Include your CSS file -->
 </head>
 <body>
   <h1>User Details</h1>
-  <div>
+  <div class="user-details">
```

Comment:
- [ ] Kindly note that the recent posts of the user should be displayed on the user show page, using the recent_posts method in the User model.

- [ ] Kindly ensure the design of the user show page matches the show page layout design.

- [ ] This page should also have a See All Posts button that takes the user to the user's posts index page.

  ![image](https://github.com/mpa-mxiang/Blog-app/assets/81468675/e1151424-6edf-43bf-bbcc-a4dede1283b2)

  ![image](https://github.com/mpa-mxiang/Blog-app/assets/81468675/8008bb4c-d735-4187-8bd4-629f853d19e2)